demo/src/lib/layout/StatusAlert.svelte

```diff
@@ -4,29 +4,31 @@
 	import biExclamationTriangleFill from 'bootstrap-icons/icons/exclamation-triangle-fill.svg?raw';
 	import {page} from '$app/stores';
 	import Svg from './Svg.svelte';
+	import {untrack} from 'svelte';
 
 	const regex = /\/(components|services)\/([^/]+)/;
 	const typeIcon: Record<string, string> = {
 		info: biInfoCircleFill,
 		warning: biExclamationTriangleFill,
 	};
 
-	export let status: string;
+	let {status}: {status: string} = $props();
 	let alert: Alert;
 
-	$: type = (status === 'inprogress' ? 'warning' : 'info') as AlertProps['type'];
-	$: statusText = status === 'inprogress' ? 'In progress' : 'Beta';
-	$: statusTextBody =
+	let type = $derived((status === 'inprogress' ? 'warning' : 'info') as AlertProps['type']);
+	let statusText = $derived(status === 'inprogress' ? 'In progress' : 'Beta');
+	let statusTextBody = $derived(
 		status === 'inprogress'
 			? 'This component is still under active development. More features will be added in the near future.'
-			: 'This component has all the basic functionalities implemented. More polishing features might be added in the near future.';
-	$: componentName = $page.url.pathname.match(regex)?.[2];
-	$: issueUrl = `https://github.com/AmadeusITGroup/AgnosUI/issues?q=is%3Aopen+is%3Aissue+label%3A%22widget%3A+${componentName}%22`;
-	$: {
+			: 'This component has all the basic functionalities implemented. More polishing features might be added in the near future.',
+	);
+	let componentName = $derived($page.url.pathname.match(regex)?.[2]);
+	let issueUrl = $derived(`https://github.com/AmadeusITGroup/AgnosUI/issues?q=is%3Aopen+is%3Aissue+label%3A%22widget%3A+${componentName}%22`);
+	$effect(() => {
 		// eslint-disable-next-line @typescript-eslint/no-unused-expressions
 		componentName;
-		alert?.api.open();
-	}
+		untrack(() => alert!.api.open());
```

Comment:
(same comment as for the other untrack usage)

Reply:
Comment added and issue on our side created !